<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareChiptunePanelWidget</class>
 <widget class="QWidget" name="HardwareChiptunePanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>696</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(48, 48, 48); 
color: rgb(208, 208, 208);
}


QWidget:disabled {
background-color:  rgb(80, 80, 80) ;
color: rgb(184, 184, 184);
}

QPushButton{
  background-color: rgb(64, 64, 64); 
 color: rgb(128, 128, 128);
}

QPushButton:checked{
  background-color: rgb(32, 32, 32); 
 color: rgb(128, 128, 128);
}
</string>
  </property>
  <widget class="QWidget" name="SongWidget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>380</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: rgb(64, 64, 64); 
color: rgb(208, 208, 208);
}</string>
   </property>
  </widget>
  <widget class="QWidget" name="TrackWidget" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>80</y>
     <width>232</width>
     <height>596</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
background-color: rgb(64, 64, 64); 
color: rgb(208, 208, 208);
}</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="TrackIndexSpinBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>64</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="displayIntegerBase">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayTrackPushButton">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>18</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>▷</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="PlayingTrackIndexLabel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>20</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="SongIndexSpinBox">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>20</y>
     <width>64</width>
     <height>48</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="displayIntegerBase">
    <number>16</number>
   </property>
  </widget>
  <widget class="QPushButton" name="PlaySongPushButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>▷</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
